### PR TITLE
Update react-native.md

### PR DESCRIPTION
**Pull request by DomiR**
Relative pod links seem to be outdated

### CODE REVIEW

**Comment by jwargo** [comment]
@DomiR thanks, I'll have the SDK team take a look at this.

**Comment by dhei** [comment]
Hi @DomiR,

This section of the doc explains how to integrate App Center SDK using relative path. In order for cocoapods to know the local relative path of App Center SDK, developers will have to specify the SDK's **podspec** file path, for example: 

`pod 'appcenter-analytics', path: '../node_modules/appcenter-analytics'` 

where the **podspec** located at **../node_modules/appcenter-analytics/ios/appcenter-analytics.podspec**

It doesn't really matter whether the path is `'../node_modules/appcenter-analytics'` OR `'../node_modules/appcenter-analytics/ios'` because they are both correct. Technically I haven't seen any reason one is better than the other.

I'd like to learn why you would prefer the `'../node_modules/appcenter-analytics/ios'` over the other.

File: docs/sdk/getting-started/react-native.md [hunk 1]
```diff
@@ -133,10 +133,10 @@ Do this integration, if you link React modules using relative path inside your P
 1. Inside your Podfile, insert the following lines:
```
Comment (ryanscottnelsen1988) [inline docs/sdk/getting-started/react-native.md:133]: Duplicate of #

File: docs/sdk/getting-started/react-native.md [hunk 2]
```diff
@@ -133,10 +133,10 @@ Do this integration, if you link React modules using relative path inside your P
...
-    pod 'appcenter-analytics', path: '../node_modules/appcenter-analytics'
-    pod 'appcenter-crashes', path: '../node_modules/appcenter-crashes'
-    pod 'appcenter-push', path: '../node_modules/appcenter-push'
+    pod 'appcenter', path: '../node_modules/appcenter/ios'
+    pod 'appcenter-analytics', path: '../node_modules/appcenter-analytics/ios'
+    pod 'appcenter-crashes', path: '../node_modules/appcenter-crashes/ios'
+    pod 'appcenter-push', path: '../node_modules/appcenter-push/ios'
     ```
 
 2. Run `pod install` from your `iOS` folder.
```
Comment (ryanscottnelsen1988) [inline docs/sdk/getting-started/react-native.md:142]:
```suggestion
2. Run `pod install` from your `iOS` folder.


```

**Review by ryanscottnelsen1988 — APPROVED** [review]

**Comment by dhei** [comment]
Hi @DomiR,

The upside of using path `../node_modules/appcenter` allows developers to use customized path for their React Native iOS project other than `{ProjectRoot}/ios`.

As I explained in my last reply, I haven't seen a compelling reason why changing the path from `../node_modules/appcenter` to `../node_modules/appcenter/ios` is necessary. So I believe it's better to keep it the current way.

Please feel free to leave a comment to let us know if you have more question or concern about it. For now I am going to close the issue.

**Comment by DomiR** [comment]
Sorry, though this was the source of my install problems... I changed that part according to some Stackoverflow posts, but at some point (after reinstalling several times) it just worked and I'm not able to tell why. Thank you for your consideration... I didn't know that pods will search/use ios folders.

**Comment by dhei** [comment]
@DomiR,

You are very welcome! 

We welcome pull requests and please keep them coming. 👍

**Pull request closed without merging**